src/prob/pf.jl

```diff
@@ -537,7 +545,7 @@ function _cpf_branch_interface(branch::Dict{String,<:Any}, v_start::Dict{<:Any,<
     Yto = branch["g_to"]+im*branch["b_to"]
     y_prim = [(Ys.+Yfr) -Ys; -Ys (Ys.+Yto)]
     bts = [[(branch["f_bus"], t) for t in branch["f_connections"]]..., [(branch["t_bus"], t) for t in branch["t_connections"]]...]
-    return  bts, 0, y_prim, nothing
+    return  bts, 0, y_prim, nothing, nothing
```

Comment:
You should always return a ccd function. if the component is linear the function should be:
```
ccd_func = function(v_bt)
                    return y_prim * v_bt 
                end
```